packages/appsync-emulator-serverless/schema.js

```diff
@@ -47,14 +47,14 @@ class AppSyncError extends Error {
 const buildVTLContext = ({ root, vars, context, info }, result = null) => {
   const {
     jwt: { iss: issuer, sub },
-    request,
+    request: { headers },
```

Comment:
Looks like request is not always set (probably somewhere in tests) I would harden this to accept a null object.